pyuntl/untldoc.py

```diff
@@ -675,3 +676,43 @@ def find_untl_errors(untl_dict, **kwargs):
         'error_dict': error_dict,
     }
     return found_data
+
+
+def untl_dict_to_tuple(untl_dict):
+    """Convert untl_dict values to list of lists of tuples."""
+    untl_tuple = {}
+    for elem in untl_dict:
```

Comment:
Now that I am looking at this again, it seems like it would make this a little neater to read if we did something like:
`for elem, values in untl_dict.items():` What do you think?

Reply:
oh..yea, sure!